index.html

```diff
@@ -1204,149 +1238,218 @@ <h1>Core Properties</h1>
 with possible differences in constraints.
     </p>
 
+    <table class="simple">
+      <thead>
+        <tr>
+          <th>Property</th>
+          <th>Required?</th>
+          <th>Value constraints</th>
+          <th>Definition</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>`id`</td>
+          <td>yes</td>
+          <td>
+A <a data-cite="INFRA#string">string</a> that conforms to rules defined
+in Section <a href="#did-syntax"></a>.
+          </td>
+          <td>
+Section [[[#did-subject]]].
+          </td>
+        </tr>
+        <tr>
+          <td>`controller`</td>
+          <td>no</td>
+          <td>
+A <a data-cite="INFRA#string">string</a> or a
+<a data-cite="INFRA#ordered-set">set</a> of
+<a data-cite="INFRA#string">strings</a>, each of which conforms to the URL
+syntax which are compatible with Section <a href="#did-syntax"></a>.
+          </td>
+          <td>
+Section [[[#did-controller]]].
+          </td>
+        </tr>
+        <tr>
+          <td>`alsoKnownAs`</td>
+          <td>no</td>
+          <td>
+A <a data-cite="INFRA#ordered-set">set</a> of
+<a data-cite="INFRA#string">strings</a>, each of which conforms to the URL
+syntax which are compatible with Section <a href="#did-syntax"></a>.
+          </td>
+          <td>
+<a data-cite="CID#also-known-as">Section 2.1.3: Also Known As</a>
+of the [[[CID]]] specification.
+          </td>
+        </tr>
+        <tr>
+          <td>`service`</td>
+          <td>no</td>
+          <td>
+A <a data-cite="INFRA#ordered-set">set</a> of [=service=]
+<a data-cite="INFRA#ordered-map">maps</a>.
+          </td>
+          <td>
+<a data-cite="CID#services">Section 2.1.4: Services</a>
+of the [[[CID]]] specification.
+          </td>
+        </tr>
+        <tr>
+          <td>`verificationMethod`</td>
+          <td>no</td>
+          <td>
+A <a data-cite="INFRA#ordered-set">set</a> of
+[=verification method=] <a data-cite="INFRA#ordered-map">maps</a>.
+          </td>
+          <td>
+<a data-cite="CID#verification-methods">Section 2.2: Verification Methods</a>
+of the [[[CID]]] specification.
+          </td>
+        </tr>
+        <tr>
+          <td>`authentication`</td>
+          <td>no</td>
+          <td>
+A <a data-cite="INFRA#ordered-set">set</a> of <a
+data-cite="INFRA#string">strings</a>, each of which conforms to
```

Comment:
Again, aren't we constraining the values to conform to the DID URL syntax? Not just any URL

> The value of the [id](https://www.w3.org/TR/did-core/#dfn-id) property for a [verification method](https://www.w3.org/TR/did-core/#dfn-verification-method) MUST be a [string](https://infra.spec.whatwg.org/#string) that conforms to the rules in Section [3.2 DID URL Syntax](https://www.w3.org/TR/did-core/#did-url-syntax).

Reply:
This applies to all verificationRelationships below aswell